Makefile

```diff
@@ -178,6 +178,18 @@ deb-loader: mig-loader
 		--architecture $(FPMARCH) -v $(BUILDREV) \
 		-s dir -t deb .
 
+rpm-loader: mig-loader
+	rm -fr tmp
+	$(INSTALL) -s -D -m 0755 $(BINDIR)/mig-loader tmp/sbin/mig-loader
+	$(INSTALL) -D -m 0644 LICENSE tmp/usr/share/doc/mig-loader/copyright
```

Comment:
This change was a lintian thing (https://lintian.debian.org/). Some package mgr frontends seem to run this and return errors if the lintian passes don't check.

This file is required in the checks so we just use that filename in this case (see #197).